### ISSUE TITLE
Role fails on a non firewalld enabled node due to `imediate: true` option.

### ISSUE THREAD

**Issue by lkusmir** [voidquark/node_exporter#1]
A running firewalld is actually a requirement for the project, as one of `Enable firewalld rule to expose node_exporter port` or `Ensure that node_exporter firewalld rule is not present`  will eventually run with the `imediate: true` option.

In an environment where `firewall-cmd --state` is `not running`, one of the above tasks will fail with the following error:
```
FAILED! => {"changed": false, "msg": "firewall is not currently running, unable to perform immediate actions without a running firewall daemon"}
``` 
depending on the `node_exporter_expose_port` value.

Steps to recreate issue:
1. Disable the firewalld on the node.
2. Set `node_exporter_expose_port` to `false`.
3. Run the role against the node.

**Comment by voidquark** [comment]
Hi,

Thx for the reporting this issue. I’ll update the role to avoid this issue. At first, I thought skipping the firewall check would be okay since I prefer not to have the firewall disabled, but it seems some people still do that.

**Comment by voidquark** [comment]
fixed by: https://github.com/voidquark/node_exporter/commit/00e63c638a3fbf855a5ef8cb359b8a53107bfb5b

**Comment by voidquark** [comment]
Closing this issue because it has been fixed in version `1.0.1`.
